app/controllers/api/application_plans_controller.rb

```diff
@@ -1,13 +1,12 @@
+# frozen_string_literal: true
+
 class Api::ApplicationPlansController < Api::PlansBaseController
 
   before_action :authorize_manage_plans, only: %i[new create destroy]
   before_action :activate_sidebar_menu
   prepend_before_action :authorize_manage_plans, only: %i[new create destroy]
 
-  with_options :only => [:index, :new, :edit, :create, :update, :destroy, :masterize] do |options|
-    options.before_action :find_service
-    options.before_action :activate_submenu
-  end
+  before_action :find_service, only: %i[index new create update destroy masterize]
```

Comment:
We don't want it for `edit` anymore. Is that correct?

Reply:
It is not correct. But no worries, I am just experimenting yet. Specially with what i am about to push now lol.